bool ask
---
float64[] done
